compiler/noirc_frontend/src/parser/parser/structs.rs

```diff
@@ -28,12 +28,14 @@ pub(super) fn struct_definition() -> impl NoirParser<TopLevelStatement> {
         .or(just(Semicolon).to(Vec::new()));
 
     attributes()
+        .or_not()
```

Comment:
```suggestion
```
Attributes should already be optional (the .repeated should be 0 or more times).